samples/msal-browser-samples/VanillaJSTestApp2.0/app/IE11/graph.js

```diff
@@ -0,0 +1,28 @@
+// Helper function to call MS Graph API endpoint 
+// using authorization bearer token scheme
+function callMSGraph(endpoint, accessToken, callback) {
+    const bearer = "Bearer " + accessToken;
+
+    var xmlHTTP = new XMLHttpRequest();
+    xmlHTTP.onreadystatechange = function () {
+        if (this.readyState == 4 && this.status == 200) {
+            callback(JSON.parse(this.responseText));
+        }
+    }
+    xmlHTTP.open("GET", endpoint, true);
+    xmlHTTP.setRequestHeader('Authorization', bearer)
+    xmlHTTP.send();
+}
+
+function seeProfileRedirect() {
+    const currentAcc = myMSALObj.getAccountByUsername(username);
```

Comment:
`const` works? (thought it didnt)

Reply:
Tested end to end and it did not complain